### DISCUSSION TITLE
Question About Base Environment

### DISCUSSION THREAD

**Discussion by wang975679801** [PaddlePaddle/PaddleSpatial#70] [BaiduKDDCup2022] [answered]
Please check whether LightGBM is installed in the base environment. I can evaluate my LightGBM model offline using the latest evaluation code, but it returns 'NoneType' object has no attribute 'forecast' when we submit the results online.

**Comment by ramber1836** [comment]
Can you provide the traceback that causes this error?

**Reply by wang975679801** [reply]
If we delete import lightgbm, the online evaluation can return results. However, once we add it in the code, the online evaluation returns:
'NoneType' object has no attribute 'forecast'.

**Comment by ramber1836** [comment, marked as answer]
Hi, there,

You can provide your team name and the administrator will reply to you in a
couple of days (due to the hard workload) by checking the online test log.
Thanks for your understanding.

Regards,
Xinjiang Lu
